packages/react-sdk/README.md

```diff
@@ -72,19 +72,34 @@ class App extends React.Component {
 }
 
 ```
+# Contents
+1. [Installation](#installation)
+2. [Usage](#usage)
+3. [Credits](#credits)
+4. [Additional code](#additional-code)
+5. [Contribute to this repo](#contribute-to-this-repo)
+
 
 # Installation
 
 ```
 npm install @optimizely/react-sdk
 ```
-
 # Usage
 
-## Creating the `ReactSDKClient`
+## `createInstance`
 
 The `ReactSDKClient` client created via `createInstance` is the programmatic API to evaluating features and experiments and tracking events. The `ReactSDKClient` is what powers the rest of the ReactSDK internally.
 
+*arguments*
+* `config : optimizely.ClientConfig` Object with SDK configuration parameters. This has the same format as the object passed to the `createInstance` method of the core `@optimizely/javascript-sdk` module. For details on this object, see the following pages from the developer docs:
```

Comment:
We don't really mention `config` or `optimizely.ClientConfig` anywhere else. From my perspective, this feels a bit confusing. Is this to support TypeScript engineers?

I think it's possible that React SDK users are not even going to think about the Optimizely JavaScript SDK because they don't have to.

Reply:
> We don't really mention config or optimizely.ClientConfig anywhere else. From my perspective, this feels a bit confusing. Is this to support TypeScript engineers?
Thanks for catching that, that's a mistake, I meant to change `optimizely.ClientConfig` to `object`.

I wasn't intentionally trying to write for TypeScript engineers, rather I needed a way to explain the types in the API/interfaces, and using TypeScript or psuedo-Typescript seemed somewhat appropriate because the code is TypeScript.